_sass/blocks/explore/_regions-list.scss

```diff
@@ -0,0 +1,74 @@
+.subregion {
+  border-top: 1px solid $mid-gray;
+  clear: both;
+  padding: 0 $base-padding;
+  position: relative;
+
+}
+
+.region-header {
+  margin-bottom: 0;
```

Comment:
Properties should be ordered border-bottom, border-top, font-weight, margin-bottom, padding-bottom, padding-top